modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -1011,4 +1081,9 @@ public void addImportsToOneOfInterface(List<Map<String, String>> imports) {
             }
         }
     }
+
+    @Override
+    public Map<String, String> typeMapping() {
```

Comment:
I dont think this override is needed here. `typeMapping()` is defined as public method in `DefaultCodeGen` and this doesnt change any behaviour. it returns the same `typeMapping` hashmap.